docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -98,15 +98,45 @@ If you do not specify either, then the DaemonSet controller will create Pods on
 
 ## How Daemon Pods are Scheduled
 
-Normally, the machine that a Pod runs on is selected by the Kubernetes scheduler.  However, Pods
+### Scheduled by DaemonSet controller (default)
+
+Normally, the machine that a Pod runs on is selected by the Kubernetes scheduler. However, Pods
 created by the DaemonSet controller have the machine already selected (`.spec.nodeName` is specified
-when the Pod is created, so it is ignored by the scheduler).  Therefore:
+when the Pod is created, so it is ignored by the scheduler). Therefore:
 
  - The [`unschedulable`](/docs/admin/node/#manual-node-administration) field of a node is not respected
    by the DaemonSet controller.
  - The DaemonSet controller can make Pods even when the scheduler has not been started, which can help cluster
    bootstrap.
 
+### Scheduled by default scheduler (with ScheduleDaemonSetPods alpha feature)
+
+In version 1.10, `ScheduleDaemonSetPods` was introduced as an alpha feature. With this feature,
+ DaemonSet Pods are scheduled by default scheduler, instead of Daemonset controller.
+
+When this feature is enabled, the `NodeAffinity` term (instead of `.spec.nodeName`) is added to the DaemonSet Pods,
+as in the following example. This enables the default scheduler to bind the Pod to the target host. If the `kubernetes.io/hostname`
+ node affinity of DaemonSet Pod already exists, it will be replaced. DaemonSet controller will only perform these operations
+ when creating DaemonSet Pods; and those operations will only modify the Pods of DaemonSet, no chagnes are made to
+ the `spec.template` of DaemonSet.
+
+```
+nodeAffinity:
+  requiredDuringSchedulingIgnoredDuringExecution:
+    nodeSelectorTerms:
+    - matchExpressions:
+      - key: kubernetes.io/hostname
+        operator: In
+        values:
+        - target-host-name
+```
+
+When this feature is enabled, `node.kubernetes.io/unschedulable:NoSchedule` toleration is added automatically to
+ DaemonSet Pods. If the `unschedulable` field of a node is not expected to be respected by the DaemonSet controller,
```

Comment:
> If the `unschedulable` field of a node is not expected to be respected by the DaemonSet controller, `TaintNodesByCondition` has to be enabled.

I think we should rewrite this sentence as:
DaemonSet controller was ignoring `unschedulable` Nodes when scheduling DaemonSet Pods. In order to ensure that default scheduler keeps the same behavior and schedules DaemonSet Pods on `unschedulable` nodes, `TaintNodesByCondition` must be enabled.

Reply:
done :)